ray-operator/Makefile

```diff
@@ -106,13 +106,29 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
+install-with-webhooks: manifests kustomize ## Install CRDs with webhooks into the K8s cluster specified in ~/.kube/config.
+	($(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone config/crd-with-webhooks | kubectl create -f -) || ($(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone config/crd-with-webhooks | kubectl replace -f -)
+
+uninstall-with-webhooks: manifests kustomize ## Uninstall CRDs with webhooks from the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone config/crd-with-webhooks | kubectl delete -f -
```

Comment:
`--load-restrictor LoadRestrictionsNone` is needed to allow kustomize to load files from outside root of its context. Otherwise, we get the error

```
make uninstall-with-webhooks

test -s /Users/dxia/src/github.com/ray-project/kuberay/ray-operator/bin/controller-gen || GOBIN=/Users/dxia/src/github.com/ray-project/kuberay/ray-operator/bin/controller-gen/.. go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.0
/Users/dxia/src/github.com/ray-project/kuberay/ray-operator/bin/controller-gen "crd:maxDescLen=0,trivialVersions=true,preserveUnknownFields=false,generateEmbeddedObjectMeta=true,allowDangerousTypes=true" rbac:roleName=kuberay-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
test -s /Users/dxia/src/github.com/ray-project/kuberay/ray-operator/bin/kustomize ||  GOBIN=/Users/dxia/src/github.com/ray-project/kuberay/ray-operator/bin/kustomize/.. go install sigs.k8s.io/kustomize/kustomize/v5@latest
/Users/dxia/src/github.com/ray-project/kuberay/ray-operator/bin/kustomize build config/crd-with-webhooks | grep -v 'creationTimestamp: "null"' | kubectl delete -f -
Error: accumulating resources: accumulation err='accumulating resources from 'bases/ray.io_rayclusters.yaml': security; file '/Users/dxia/src/github.com/ray-project/kuberay/ray-operator/config/crd-with-webhooks/bases/ray.io_rayclusters.yaml' is not in or below '/Users/dxia/src/github.com/ray-project/kuberay/ray-operator/config/crd-with-webhooks'': must build at directory: '/Users/dxia/src/github.com/ray-project/kuberay/ray-operator/config/crd-with-webhooks/bases/ray.io_rayclusters.yaml': file is not directory
No resources found
make: *** [uninstall-with-webhooks] Error 1
```